commcare-support-library/src/main/java/org/commcare/commcaresupportlibrary/identity/identity_integration.md

```diff
@@ -20,6 +20,12 @@ Often you will need this generated guid to be passed back to CommCare so that it
 IdentityResponseBuilder.registrationResponse(guid)
     .finalizeResponse(activity)
 ````
+Alternatively, in case the biometric templates are to be stored in CommCare, use the following instead -
+````
+IdentityResponseBuilder.registrationResponse(guid, templates)
+    .finalizeResponse(activity)
+````
+* `templates` is a [`Map`](https://docs.oracle.com/javase/8/docs/api/java/util/Map.html) containing all the biometric templates and whose _keys_ are [`BiometricIdentifier`](BiometricIdentifier.java) elements and _values_ are the actual biometric templates in the form of a byte array

```

Comment:
We should add a sample code below this to give an example of how to construct templates using Biometric indentifiers.